tasker/job_template/generate.py

```diff
@@ -0,0 +1,36 @@
+from tasker.models import db, User, JobTemplate, Task, TaskStatus
+import datetime
+
+def generate_tasks(template_id):
+    template = JobTemplate.query.get(template_id)
+    user = template.owner
+    rep = template.repetition
+    interval = template.interval
+    if interval == 2:
+        rep *= 7
+    if interval == 3:
+        rep *= 30
+    start = template.starting_date
+    timestamp = datetime.datetime.fromtimestamp(start)
```

Comment:
So, `start` from the Job Template is an epoch datetime.  I think we should make the timestamp, like you're doing here and then change the hour to `template.hour` and minute to 0.  Then localize it to the user's timezone with `pytz`.

Reply:
The rest of the logic for adding timedeltas looks great to me.